x/wasm/keeper/options.go

```diff
@@ -81,3 +81,26 @@ func WithVMCacheMetrics(r prometheus.Registerer) Option {
 		NewWasmVMMetricsCollector(k.wasmVM).Register(r)
 	})
 }
+
+// WithCosts sets custom gas costs and multiplier.
+// See DefaultCompileCost, DefaultInstanceCost, DefaultGasMultiplier
+// Uses WithApiCosts with defaults and given multiplier.
+func WithCosts(compile, instance, multiplier uint64) Option {
```

Comment:
I like this as cleaner than global vars.

However, just want to point out, these options could be applied manually after construction, so don't prevent malicious usage of them (but make that more obvious)

`WithCosts(1, 2, 3)(keeper)` can be called anywhere.

Reply:
The [`apply` method](https://github.com/CosmWasm/wasmd/blob/master/x/wasm/keeper/keeper.go#L44) is package private. We are safe with this.